package.json

```diff
@@ -14,6 +14,9 @@
     "airbrake",
     "notifier"
   ],
+  "dependencies": {
+    "isomorphic-fetch": "^2.2.1"
```

Comment:
I don't see the point of adding the dependency here. Given that it should be up to the consumer having this available.

Reply:
hold on with that, it looks like isomorphic-fetch is now required? let me double check

Reply:
You latest commit makes this PR very similar to mine :-)
I am adding any further comment in the issue #465

Reply:
My thinking is that most people don't have/want to deal with Webpack configuration so making their life easier is better than making bundle smaller + forcing them to learn how to configure Webpack. And some of them simply don't use Webpack at all...

Reply:
Good thinking. That works for me